cards.tex

```diff
@@ -135,6 +155,8 @@ \section{Frightened}
 \foreach[evaluate=\level as \bonus using int(\level*8)] \level in {1, ..., 10} {
 \Cantrip{\level}{Ray of Frost}
 
+\Traits{\Trait{Attack} \Trait{Cantrip} \Trait{Cold} \Trait{Evocation}}
```

Comment:
"attack" and "cantrip" are already somewhere else on the card, do we want to repeat it here?

My thinking was to put an explicit "Cast" action on each spell card, and split the traits somehow cleverly between the spell itself and the action to cast it.

There is some complicated stuff going on with spell traits though. For example, if a spell needs [somatic](https://2e.aonprd.com/Rules.aspx?ID=283) components, the action to cast it gains the [manipulate](https://2e.aonprd.com/Traits.aspx?ID=104) trait. So should we put the words "somatic", "manipulate", or both on the card?

Reply:
For most things we try to be more explicity and include the rule, instead of relying on the player to remember the rule. So, I think spells with _somatic_ component should also have the _manipulate_ trait tag on the card. 

Having one tag row is a good start, we can easily split / distribute these tags over multiple places (like individual spells or attacks). Definitely needs more formatting, though 🙈